go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/k8snetworkplumbingwg/sriov-network-device-plugin
 
-go 1.22.4
+go 1.23.4
```

Comment:
for golang version bump need to update workflows and Dockerfiles under images as well. currently they point to 1.22

Reply:
done